### ISSUE TITLE
The recent change that removes "Night" mode affects other security system plugins

### ISSUE THREAD

**Issue by alxalxnet** [dgreif/ring#344]
# Bug Report

### Describe the Bug
 The latest version of homebrodge-ring removes "Night" mode also from other plugins (e.g., homebridge-securitysystem)

### To Reproduce
I use homebridge-ring only to view my cameras and I set up a complex system of rules for my security system with homebridge-securitysystem, for which I need also the Night mode. The latest version of homebridge-ring removes the Nightmode for homebridge-securitysystem.

### Expected behavior
 Homebridge-ring shouldn't interfere with other plugins.

### Screenshots/Logs
 If applicable, add screenshots and log output to help explain your problem.

### Additional context
 Add any other context about the problem here.

### Environment

 - OS: [e.g. Raspbian]
 - Node.js: [e.g. 12.14.1] 
 - NPM: [e.g 6.13.4] 
 - ring-client-api: [e.g 6.0.3]

**Comment by dgreif** [comment]
The way that services/characteristics are designed in homebridge should prevent any cross-plugin or cross-accessory issues like this.  `homebridge-securitysystem` does appear to support disabling specific modes, so maybe check your config and see if you have `disabled_modes` set in that plugin.  https://github.com/MiguelRipoll23/homebridge-securitysystem/tree/458fce1509c106fe23c3c22910f782f0bba5e7d6#basic-options

**Comment by dgreif** [comment]
Any updates? Did you figure it out in the other plugin?

**Comment by alxalxnet** [comment]
Sorry for the delay.

Yes, I figured it out. The issue was that the other plugin created an alarm that I called "Security System" and when the alarm feature was added to homebridge-ring, it was automatically registered as Security System. I tried many times to disable the homebridge-ring alarm system, but I couldn't find a way to do that. So the two co-existed without a problem, until you removed the Night option from homebridge-ring, which removed it also on homebridge-security and broke all the automations.
It's all good now, and I gave each system a different name, but let me know if there's a way to disable the ring security system completely.

Thanks.

Alex
